website/docs/en/contribute/development/releasing.md

```diff
@@ -57,7 +57,9 @@ git push origin release/vx.y.z
 
 4. Create a PR with the title `chore: release v1.0.0`.
 5. Run the [Ecosystem CI workflow](https://github.com/web-infra-dev/rspack/actions/workflows/ecosystem-ci.yml) to ensure all ecosystem projects are working properly.
-6. Run the full release workflow on the release branch.
+6. Run the full release workflow on the release branch:
+   - [Release Full(OTP)](https://github.com/web-infra-dev/rspack/actions/workflows/release-otp.yml): Publish NPM packages to registry
```

Comment:
Ref: [NPM should be lower case](https://github.com/nodejs/nodejs.org/issues/6917)

```suggestion
   - [Release Full(OTP)](https://github.com/web-infra-dev/rspack/actions/workflows/release-otp.yml): Publish npm packages to registry
```